src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import glamorous from 'glamorous';
+
+import Button from '../Button/Button';
+import constants from '../../constants';
+
+const { borderRadius } = constants;
+
+const ButtonGroupContainer = glamorous.div({
+   display: 'inline-block',
+});
+
+const GroupedButton = glamorous(Button)({
+   borderRadius: 0,
+
+   '&:first-of-type': {
+      borderTopLeftRadius: borderRadius,
+      borderBottomLeftRadius: borderRadius,
+   },
+   '&:last-of-type': {
+      borderTopRightRadius: borderRadius,
+      borderBottomRightRadius: borderRadius,
+   },
+});
+
+const ButtonGroup = ({
+   className,
+   disabled,
+   onMouseEnter,
+   onMouseLeave,
+   children,
+}) => (
+   <ButtonGroupContainer
+      className={className}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+   >
+      {React.Children.map(children, child => {
+         if (child.type !== Button) {
+            return child;
```

Comment:
I did this because, in my use case, I wanted to put a hidden input inside of the ButtonGroup without it becoming a Button.  I figured this was fine and it would be better not to screw with children that aren't buttons but maybe I'm wrong on that.

Reply:
I was initially against this approach but after thinking about it for a little bit I think this is probably the right way to go. It'll prevent the `ButtonGroup` component from breaking if it's passed components that aren't `Button`s 👍